src/container/cli.c

```diff
@@ -331,110 +331,62 @@ dc_cont_destroy(tse_task_t *task)
 	struct dc_pool		*pool;
 	crt_endpoint_t		 ep;
 	crt_rpc_t		*rpc;
+	crt_opcode_t		 opc;
 	struct cont_args	 arg;
+	bool			 by_label;
+	bool			 by_uuid;
 	int			 rc;
 
 	args = dc_task_get_args(task);
 
-	if (uuid_is_null(args->uuid))
+	/* destroy by one: UUID or label */
+	by_uuid = daos_uuid_valid(args->uuid);
+	by_label = (bool)(args->label != NULL);
+	if (by_uuid == by_label)
```

Comment:
Minor: it would be helpful here (and in open) to log an error here that you can only use 1

Reply:
Agree that would be much better. Added to latest code on the branch.